sources/packages/backend/apps/api/src/app.system-access.module.ts

```diff
@@ -43,6 +43,7 @@ import {
   ECertIntegrationSystemAccessController,
   FedRestrictionsIntegrationSystemAccessController,
   ATBCSystemAccessController,
+  IERIntegrationSystemAccessController,
```

Comment:
Why is this not using a module created in shared library? Story was asking for `Create module in shared library`